lib/client/browser.js

```diff
@@ -219,6 +219,16 @@ require("./discovery.js");
 					});
 
 					QUnit.log(function(details) {
+						if (config.logAllAssersions) {
```

Comment:
```suggestion
						if (config.logAssertions) {
```
I think `all` is not required here, as by default no assertions are logged. Only the tests.